packages/ui5-application-writer/templates/optional/typescript/package.json

```diff
@@ -1,5 +1,7 @@
 {
     "scripts": {
+        "prestart": "npm run ts-typecheck",
+        "prebuild": "npm run ts-typecheck",
```

Comment:
@jlongie as discussed, can we limit this to the FF SAPUI5 floorplan only for now?

Reply:
Addressed in latest PR.

Reply:
@devinea could you please explain why you would only do it for the basic template?

Reply:
See https://github.com/SAP/open-ux-tools/pull/747#discussion_r1000661047